tests/bootstrap.php

```diff
@@ -0,0 +1,2 @@
+<?php
+require_once(__DIR__ . '/../vendor/autoload.php');
```

Comment:
I prefer this style of writing paths:

    require_once(dirname(__DIR__) . '/vendor/autoload.php');